Makefile

```diff
@@ -63,6 +63,14 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
+NOTICE: vendor/
+	rm -f NOTICE # `govendor license` appends...
+	govendor license -o NOTICE +v
+
+notice-up-to-date: NOTICE
```

Comment:
Since it is a test maybe prepend it name with test- ? and add .PHONY just in case ?